electron/main.js

```diff
@@ -68,6 +68,12 @@ if (config.DEVELOPMENT) {
   app.commandLine.appendSwitch('ignore-certificate-errors', 'true');
 }
 
+if (argv.portable) {
+  const EXEC_PATH = (process.env.APPIMAGE != null) ? process.env.APPIMAGE : process.execPath;
+  const USER_PATH = path.join(EXEC_PATH, '..', 'WireData');
```

Comment:
This will generate the `WireData` directory next to `Wire`.. right?!

I was thinking.. why not having something like that: 

```js
const USER_PATH = path.join(EXEC_PATH, 'Data');
```